zyte_common_items/pipelines.py

```diff
@@ -2,6 +2,8 @@
 
 from zyte_common_items import ae
 
+DEFAULT_ITEM_PROBABILITY_THRESHOLD = 0.1
```

Comment:
Making it a pipeline attribute would make it easier to customize a pipeline - inherit from it, change a value.